plugins/easyimage/plugin.js

```diff
@@ -100,6 +105,10 @@
 				allowedContent: {
 					figure: {
 						classes: config.easyimage_sideClass
+					},
+
+					img: {
+						attributes: '!src,srcset,alt,width,height'
```

Comment:
This plugin should not add ACF rule allowing `[height,width]` rule for `img` as it doesn't use it for anything.